### ISSUE TITLE
[BITV] Files list breadcrumbs contain not valid attributes

### ISSUE THREAD

**Issue by susnux** [nextcloud/server#41780]
The breadcrumbs of the file list contain a `dir` attribute on the `li` element.
But the `dir` attribute is meant for text direction and might only have the values `ltr` `rtl` or `auto`.

This is an accessibility issue as the attribute probably should be either a `data-dir` attribute or is a vue property not correctly set.
